include/af/algorithm.h

```diff
@@ -141,6 +278,24 @@ namespace af
     */
     AFAPI array count(const array &in, const int dim = -1);
 
+#if AF_API_VERSION >= 37
+    /**
+       C++ Interface for counting non-zero values in an array according to a key
+
+       \param[out] keys_out will contain the reduced keys in \p vals along \p dim
+       \param[out] vals_out will contain the count of all values in \p vals along \p dim according to \p keys
+       \param[in] keys is the key array
+       \param[in] vals is the array containing the values to be reduced
+       \param[in] dim The dimension along which the add operation occurs
```

Comment:
"add" should be "count", or change the whole "add operation" to "counting" so it reads "along which the counting occurs"